aces/Loris
# Make the number of days since last login before making a user inactive a config

## Brief summary of changes

This adds a new Config in the config module to allow projects to configure the number of days since a user last logged in before making their account inactive.

#### Testing instructions

1. Run the SQL patch `SQL/New_patches/2023-02-28_create_max_days_inactive_config_for_users.sql`
2. Create a new user if needed and login as that user
3. For that new user, update the last login to 3 years ago `UPDATE user_login_history SET Login_timestamp='2020-02-28 19:28:35' WHERE userID=<new user id>`
4. Try logging in again with that user. You should see the following:
![Screen Shot 2023-02-28 at 2 32 22 PM](https://user-images.githubusercontent.com/1402456/221959328-1a2287c1-0f17-40d3-a75d-cb364c8849da.png)
